src/PAModel/PAConvert/Yaml/YamlLexer.cs

```diff
@@ -726,7 +726,7 @@ internal YamlToken CheckDuplicate(string propName, int currentLine)
                 if (_previousProperties.TryGetValue(propName, out oldLine))
                 {
                     // Key is already present.
-                    return YamlToken.NewError(default(SourceLocation), $"Property '{propName}' is already defined on line {oldLine}.");
```

Comment:
Can you include both the old & new line? Maybe something like "Property name duplicated on lines {oldLine} and {newLine}"?

Reply:
Done! :)